java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2862,4 +2863,19 @@ protected ArgsArray buildArgs(String... stringArgs) {
 
         return commandArgs.build();
     }
+
+    /**
+     * Appends a value to a key. If `key` does not exist it is created and set as an empty string, so
+     * `APPEND` will be similar to {@see #set} in this special case.
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/append/">redis.io</a> for details.
+     * @param key The <code>key</code> to which the value will be appended.
```

Comment:
```suggestion
     * @param key The key of the string.
```
docs should match with the interface